keyboards/rastersoft/minitkl/readme.md

```diff
@@ -0,0 +1,21 @@
+# MiniTKL
+
+![MiniTKL](https://gitlab.com/rastersoft/full-ten-keyless/-/raw/main/layout.png)
```

Comment:
please, rehost on imgur.

Reply:
@drashna Is it absolutely required? It wants my phone number, and I'm eager to put it on the internet. Is there an alternative place?

Reply:
Ok, moved to imgbox.com.